alma/composer.lock

```diff
@@ -8,16 +8,16 @@
     "packages": [
         {
             "name": "alma/alma-php-client",
-            "version": "v1.5.4",
+            "version": "v1.6.0",
```

Comment:
composer.lock should be removed from the repo.
you can `git remove` it and add it to `.gitignore` 😉